cli/command/container/opts.go

```diff
@@ -198,6 +199,7 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.BoolVar(&copts.privileged, "privileged", false, "Give extended privileges to this container")
 	flags.Var(&copts.securityOpt, "security-opt", "Security Options")
 	flags.StringVar(&copts.usernsMode, "userns", "", "User namespace to use")
```

Comment:
Looks like we missed an annotation for this one as well; I'll open a separate PR for that

Reply:
oh, never mind, this was quite old